Libraries/Text/RCTTextFieldManager.m

```diff
@@ -39,8 +39,16 @@ - (BOOL)textField:(RCTTextField *)textField shouldChangeCharactersInRange:(NSRan
   }
 
   if (textField.maxLength == nil || [string isEqualToString:@"\n"]) {  // Make sure forms can be submitted via return
+    if (textField.autocorrectionType == UITextAutocorrectionTypeYes) {
```

Comment:
What I'm confused about is the name of this function is `shouldChangeCharactersInRange` and based on that I'd expect it to return a `YES` or `NO`. But here we override the text (side effect): `textField.text =`, and then return `NO`.